### PR TITLE
add lab3 code

### PR DESCRIPTION
**Pull request by IvanVoloshanskyi**

### CODE REVIEW

File: 3.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#include<stdio.h>
+#include<string.h>
+
+
+void reverse(char arr[])
```
[the remark this answers is not in the document]
Comment (IvanVoloshanskyi) [reply 3.c:5]: Done

```suggestion
void reverse_array(char arr[])
```

[the remark this answers is not in the document]
Comment (IvanVoloshanskyi) [reply 3.c:5]: Done

File: 3.c [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+int n = strlen(arr);
```
Comment (zveres-ss) [inline 3.c:7]: what n stands for?

File: 3.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (zveres-ss) [reply 3.c:5]: reverse is totally ok - it reverses the passed array

**Pull request merged**